addons/info/README.md

```diff
@@ -78,8 +78,9 @@ import { setDefaults } from '@storybook/addon-info';
 
 // addon-info
 setDefaults({
-  inline: true,
-  maxPropsIntoLine: 1,
+  header: false, // Toggles display of header with component name and description
+  inline: true, // Displays info inline vs click button to view
```

Comment:
@tenshiemi, thank you for pointing us to this lack of documentation!

There're some more options:
`source`, `propTables`, `propTablesExclude`, `styles`

Could you also add them if you want?